test/test_event_handler_sanitizer_report.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Open Source Robotics Foundation
+# Licensed under the Apache License, Version 2.0
+
+
+from colcon_core.event.job import JobEnded
+from colcon_sanitizer_reports.event_handlers.sanitizer_report \
+    import SanitizerReportEventHandler
+from mock import patch
+
+
+def test_event_handler_asan_report():
```

Comment:
Let's make sure this gets executed as there is no main. I guess the test framework discovers and run this test, but let's double check :)

Reply:
I'll look into setting up CI for this repo that will run the tests.